purchase_work_acceptance/models/purchase.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.addons import decimal_precision as dp
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    wa_count = fields.Integer(
+        compute='_compute_wa',
+        string='WA count',
+        default=0,
+        store=True
+    )
+    wa_ids = fields.Many2many(
+        comodel_name='work.acceptance',
+        compute='_compute_wa',
+        string='Work Acceptances',
+        copy=False,
```

Comment:
I think normally, compute field do not need `copy`
Also, do we need to make a store=True?